code/modules/mob/living/carbon/human/species/necromorph/ubermorph.dm

```diff
@@ -277,7 +277,7 @@ Best used near the end, when all seems quiet, to help the necromorphs hunt down
 			//Find target organ should never fail, we won't bother checking
 			targetstring = "[H]'s [found_organ]"
 
-			playsound(charge., 'sound/weapons/slice.ogg', VOLUME_MAX, 1)
```

Comment:
good catch, wrong fix. This sound should be played from charge.victim

charge is a non atomic datum here